packages/flutter/test/foundation/diagnostics_test.dart

```diff
@@ -12,7 +10,7 @@ import 'package:flutter_test/flutter_test.dart';
 
 class TestTree extends Object with DiagnosticableTreeMixin {
   TestTree({
-    this.name,
+    this.name = '',
     this.style,
```

Comment:
This could also default to `DiagnosticsTreeStyle.none`, and be non-nullable.

Reply:
Same as the other one (https://github.com/flutter/flutter/pull/62616#discussion_r498576029)